courses/machine_learning/deepdive2/supplemental/solutions/object_detection_with_automl.ipynb

```diff
@@ -0,0 +1,559 @@
+{
```

Comment:
<ul><li>remove <code>from google.cloud import automl</code> </li><li>remove <code>dataset_id = "IOD7192557567678087168"&nbsp;</code>  and use the variable <code>dataset_id</code> directly (you already set it earlier)</li><li>Reuse <code>DATASET_NAME</code>  instead of redefining the variable <code>display_name</code></li><li> remove <code>client = automl.AutoMlClient()</code></li><li> remove <code>project_location = client.location_path(project_id, "us-central1")</code></li><li> <code>metadata</code>  -&gt; <code>training_metadata</code>  (you already defined before a variable named <code>metadata</code>)</li><li> <code>response</code>  -&gt; <code>training_results</code></li></ul>
<br/>

 _Reply via <a href='https://app.reviewnb.com/GoogleCloudPlatform/training-data-analyst/pull/1065/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='25'/>

Reply:
<p>done</p>